style.css

```diff
@@ -0,0 +1,64 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background: url(./assets/backgrounds/header-background-mobile.png);
+  background-color: #e5e5e5;
+  flex-direction: column;
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  margin: 24, 206, 24, 362;
+}
```

Comment:
- [ ] Please add more left and right padding to all page elements to match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170544244-19114476-c53e-405f-b919-6eebf2c21cf5.png) | ![image](https://user-images.githubusercontent.com/68796232/170544488-3eef28c1-493a-4eae-8272-3ecc707a44f0.png)|